makefiles/suit.v3.base.inc.mk

```diff
@@ -0,0 +1,40 @@
+#
+# path to suit-tool
+SUIT_TOOL ?= $(RIOTBASE)/dist/tools/suit_v3/suit-manifest-generator/bin/suit-tool
+
+#
+# SUIT encryption keys
+#
+
+# Specify key to use.
+# Will use $(SUIT_KEY_DIR)/$(SUIT_KEY) $(SUIT_KEY_DIR)/$(SUIT_KEY).pub as
```

Comment:
comment outdated, only `$(SUIT_KEY_DIR)/$(SUIT_KEY).pem` will be used.

Reply:
Correct, fixed in the newest fixup commit